jupyter-docker/Dockerfile

```diff
@@ -72,7 +72,7 @@ ENV HAILJAR hail-0.1-$HAILHASH-Spark-$SPARK_VER.jar
 ENV HAILPYTHON hail-0.1-$HAILHASH.zip
 ENV HAILZIP Hail-0.1-$HAILHASH-Spark-$SPARK_VER.zip
 ENV HAIL_HOME /etc/hail
-ENV KERNELSPEC_HOME /usr/local/share/jupyter/kernals
+ENV KERNELSPEC_HOME /usr/local/share/jupyter/kernels
```

Comment:
Given we haven't been getting failures with the typo in the file path, I suspect `KERNELSPEC_HOME` hasn't been used but couldn't be certain.

Reply:
possibly because there is another `KERNELSPEC_HOME` defined in `init-actions.sh`